DfE.FindInformationAcademiesTrusts/Services/ExportService.cs

```diff
@@ -0,0 +1,100 @@
+﻿using ClosedXML.Excel;
+using DfE.FindInformationAcademiesTrusts.Data;
+using DfE.FindInformationAcademiesTrusts.Pages;
+
+namespace DfE.FindInformationAcademiesTrusts.Services
+{
+    public interface IExportService
+    {
+        byte[] ExportAcademiesToSpreadsheetUsingProvider(Data.Trust? trust, Trust.TrustSummaryServiceModel? trustSummary);
+    }
+
+    public class ExportService : IExportService
+    {
+
+        public byte[] ExportAcademiesToSpreadsheetUsingProvider(Data.Trust? trust, Trust.TrustSummaryServiceModel? trustSummary)
+        {
+            var headers = new List<string>
+            {
+                "School Name", "URN", "Local Authority", "Type", "Rural or Urban", "Date joined",
+                "Previous Ofsted Rating", "Before/After Joining","Date of Previous Ofsted",
+                "Current Ofsted Rating", "Before/After Joining", "Date of Current Ofsted",
+                "Phase of Education", "Pupil Numbers", "Capacity", "% Full", "Pupils eligible for Free School Meals"
+            };
+
+            var academies = trust?.Academies ?? [];
+
+            var dataExtractor = new Func<Data.Academy, object[]>((academy) =>
```

Comment:
Using an `object[]` feels a bit icky. As most of the objects are actually strings here and they're all `ToString()`ed when they're used later (`rowData[j].ToString()`), I think we should make it a `string[]` here to force thinking about the string representation of each object at this point.

Reply:
Agree, better for type safety, done